frameworks/helloworld/cli/dcos-hello-world/main.go

```diff
@@ -2,17 +2,18 @@ package main
 
 import (
 	"github.com/mesosphere/dcos-commons/cli"
+    "github.com/mesosphere/dcos-commons/cli/commands"
 	"gopkg.in/alecthomas/kingpin.v2"
 )
 
 func main() {
 	app := cli.New()
 
-	cli.HandleConfigSection(app)
-	cli.HandleEndpointsSection(app)
-	cli.HandlePlanSection(app)
-	cli.HandlePodsSection(app)
-	cli.HandleStateSection(app)
+	commands.HandleConfigSection(app)
```

Comment:
I was only doing this for the example `helloworld` CLI - what's our policy about updating CLIs?